src/main.rs

```diff
@@ -1,19 +1,37 @@
 use anyhow;
+use clap::Clap;
 use log;
 use simplelog::{Config, LevelFilter, TermLogger, TerminalMode};
 
+mod cli;
 mod settings;
 use settings::Settings;
 
+use crate::cli::Opts;
+
 fn main() -> anyhow::Result<()> {
+    let opts = Opts::parse();
+    let setings = Settings::new(opts)?;
+
     TermLogger::init(
```

Comment:
@FridgeSeal what do you think of having specified log levels in the CLI? 
They're currently alongside the other app settings like data path and peer source. If we make them only modifiable from the CLI then i'll just use them from `opts` instead and can decide the log level before we instantiate settings

Reply:
Made up my own mind when i looked at the screenshot that had a `APP_QUIET=true` environment variable 🤦

Reply:
Hahahaha